content/en/network_performance_monitoring/devices/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Network Device Monitoring
+kind: documentation
+disable_toc: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+  tag: "Blog"
+  text: "Monitor SNMP with Datadog"
+- link: "https://www.datadoghq.com/blog/monitor-meraki/"
+  tag: "Blog"
+  text: "Monitor Cisco Meraki with Datadog"
+---
+
+## Overview
+
+Datadog Network Device Monitoring is designed to give you visibility into your network-connected devices, such as routers, switches, servers, and firewalls with Simple Network Management Protocol (SNMP).
```

Comment:
can we remove the with Simple Network Management Protocol, its an implementation detail that I think isn't necessary to lead with.